### ISSUE TITLE
[ERROR MSG FEATURE] PPSSPP SAVES (Including Save states) load but do not overwrite (or create new save) for Windows

### ISSUE THREAD

**Issue by Okwizi** [hrydgard/ppsspp#16261]
### What should happen

I recently had an issue saving my games on both the memory stick and as a SAVE STATE. I was also unable to create a new SAVE STATE for any of my existing games. I thought the save files were corrupted but I was still able to load the games. Then I realized that **Windows Security** had restricted access to where my saves are (_...\Documents\PPSSPP\PSP\SYSTEM_) The screenshot below shows the details. The "**Allow device**" option fixes this issue!
![Screenshot (18)](https://user-images.githubusercontent.com/84937129/196922903-bf34c98f-53d8-4b1d-a196-992c19bdb081.png)
Looks like **Folder Control Access** in Windows Security limits manipulation in the Documents directory if the setting is on and hence prevents PPSSPP from saving in the memory stick or as a SAVE STATE. I propose a new way of specifying the _"reason"_ why a SAVE file might not save due to the issue I have just stated above. I have taken my time and gone through existing Issues to see if this is something that had been covered in the past. If it had been then I apologize but to make PPSSPP more fun and easy to use, I'd appreciate having a bypass to Windows Security features like **Folder Control Access** or an error message stating why PPSSPP loads games but can't save them.

### Who would this benefit

This would benefit any Windows user/Windows emulator user. This will also help those who aren't into Computer Intelligence but play emulated PSP games with PPSSPP.

### Platform (if relevant)

Windows

### Games this would be useful in

ULUS104230000 - Warriors Orochi 2, ULES00192001 - SSX ON TOUR, 

### Other emulators or software with a similar feature

1. uTorrent has a feature thats notifies users when magnet files have been "**Quarantined**" by an antivirus or Windows Security.
2. DSHMC (PS3 controller driver support software) notifies users that starting the application without Administrative rights will prevent the software from properly managing and syncing Bluetooth with the PS3 controller. Also notifies that administrative rights is needed for it to run as a service in Windows.

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by ANR2ME** [comment]
I guess Documents folder can be restricted like that as it may contains sensitive data, may be PPSSPP should use folders outside Documents but still within `%userprofile%` folder

**Comment by Okwizi** [comment]
> 

You're right. I'm not entirely sure why this specific Windows Security feature affected me just recently because, before this, I could save PPSSPP games just fine. Windows 11 does have concurrent and sometimes new updates, especially with the antivirus tool so they must have updated the algorithms on the software to restrict access to selected folders. I also think the document content of PPSSPP that is kept in the default Documents folder can be moved to someplace without Administration Elevation. Another option would be to run PPSSPP as an Administrator. ( Haven't tried this but I bet it would work). And including a readme for anyone with this particular problem.
